app/config/firebase.ts

```diff
@@ -1,22 +1,10 @@
 import { initializeApp } from 'firebase/app';
 import {
   type Auth,
-  type User,
-  getAuth,
-  signInWithRedirect,
-  GoogleAuthProvider,
-  GithubAuthProvider,
+  getAuth
 } from 'firebase/auth';
 import type { FirebaseConfig } from '../types/firebase';
-import { getFirestore, doc, getDoc, setDoc } from 'firebase/firestore';
-
-interface ReloadUser {
- screenName: string
-}
-
-interface ExtendedUser extends User {
-  reloadUserInfo?: ReloadUser;
-}
+import { getFirestore, doc, getDoc, setDoc, serverTimestamp } from 'firebase/firestore';
```

Comment:
Since the db functionality now lives in the signup form I don't think you need to keep this import anymore.